src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/pages/CommonPageElements.java

```diff
@@ -0,0 +1,195 @@
+package com.marizueva.laboratory.hw3.fluentpageobject.pages;
+
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class CommonPageElements extends BasePage {
+    ServiceDifferentElementPage serviceDifferentElementPage;
+
+    @FindBy(xpath = "//li[@class='dropdown uui-profile-menu']//span[@class='caret'][1]")
+    protected WebElement openLoginDropdownButton;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::input[@id='name']")
+    protected WebElement loginField;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::input[@id='password']")
+    protected WebElement passwordField;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::span[text()='Enter']")
+    protected WebElement enterCredentials;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::span[@id='user-name']")
```

Comment:
I suppose that `@FindBy(id = "")` could be used instead of **xpath**

Reply:
done